README.md

```diff
@@ -22,3 +22,26 @@ When you’re done working in the virtual environment, simply type:
 ```
 exit
 ```
+
+To add missing package to the package list:
+```
+poetry add your_new_package_name
+```
+
+## Discord Bot Development
+
+### Getting started
+- [Create a bot in discord](https://interactions-py.github.io/interactions.py/Guides/02%20Creating%20Your%20Bot/)
+- Create your own discord server, add that bot to a test server (feel free to use )
+- Implement bot logics (and add bot token to your code too)
+- Now, run the bot code locally and test the bot on your server
+
+### Auto formatter for Python
+We use [Ruff](https://github.com/astral-sh/ruff) for linting and code formating. Ruff is already included and installed when you use `poetry install`. Follow the [instructions here](https://github.com/astral-sh/ruff) to use Ruff.
```

Comment:
```suggestion
We use [Ruff](https://github.com/astral-sh/ruff) for linting and code formating and is installed via `poetry`. Follow the [instructions here](https://github.com/astral-sh/ruff) to use Ruff.
```

Reply:
Probably should just remove. Linting should go into CONTRIBUTING.md